statsmodels/tsa/statespace/tests/test_sarimax.py

```diff
@@ -659,10 +659,9 @@ def setup_class(cls):
 
     def test_mle(self):
         result = self.model.fit(disp=-1)
-        assert_allclose(
-            result.params, self.result.params,
-            atol=1e-2, rtol=1e-3
-        )
+        # Use ratio to make atol more meaningful parameter scale differs
```

Comment:
That's a bit a strange way, rtol handles relative error, i.e. corresponds to atol in ratio already, so the main difference is in reducing rtol to 1e-2 and add a rtol on ratio.

What I do in many cases if there is a known systematic difference in scaling is to use a correction factor
`assert_allclose(result.params, self.result.params * corr_factor, ...)`